libs/razorpay/src/services/razorpay-webhook/service.ts

```diff
@@ -0,0 +1,77 @@
+import * as CryptoJS from 'crypto-js';
+
+import { getPrismaClient } from '@vestido-ecommerce/models';
+
+import { RazorpayWebhookSchemaType } from './zod';
+
+export async function handleRazorpayWebhook(data: RazorpayWebhookSchemaType) {
+  const prisma = getPrismaClient();
+  const secret = '6d2Lv76AugGojJGt1Wa7MEgFZmSZaq5z';
+
+  const webhookSignature = data.signature;
+  delete (data as { signature?: string }).signature;
+
+  const dataString = JSON.stringify(data);
+
+  const generatedSignature = CryptoJS.HmacSHA256(dataString, secret).toString(
+    CryptoJS.enc.Hex,
+  );
```

Comment:
Extract this out to libs/razorpay/src/services/signature.ts
Copy can be found at libs/razorpay/src/services/verify-signature/service.ts

Reply:
Done